plugins/chat/assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -187,11 +187,19 @@ export default class ChatLivePane extends Component {
           this.scrollToMessage(findArgs["targetMessageId"], {
             highlight: true,
           });
-        } else if (fetchingFromLastRead) {
+          return;
+        }
+
+        if (
+          fetchingFromLastRead &&
+          messages.length &&
+          findArgs["targetMessageId"] !== messages[messages.length - 1].id
```

Comment:
This is where we check if want the latest message (and a result bottom of the screen) or if we want a specific message  before the latest